### PR TITLE
Simple Write Batching

### PR DESCRIPTION
**Pull request by benbjohnson**
## Overview

This pull request adds a simple per-request batching to `Server.WriteSeries()`. It simply writes each point in the batch individually but does them all in parallel.

This is simply a stop gap solution until we implement full batching.

### CODE REVIEW

**Comment by otoolep** [comment]
Ship it.

File: server.go [hunk 1]
```diff
@@ -1331,6 +1319,49 @@ func (s *Server) WriteSeries(database, retentionPolicy string, points []Point) (
...
+	for i := range points {
+		wg.Add(1)
+		go func(p *Point) {
+			index, err := s.writePoint(database, retentionPolicy, p)
+			ch <- resp{index, err}
+			wg.Done()
+		}(&points[i])
+	}
+	wg.Wait()
+	close(ch)
```
Comment (freeformz) [inline server.go]: Probably not super important, but you could fire off a these two lines in another go routine and start the processing of the response channel below while the points are being written.

Comment (benbjohnson) [inline server.go]: That's a good point. This is just a stop gap solution while we work on more robust `Server`-level batching.

**Comment by otoolep** [comment]
@benbjohnson -- this is an important change, and @pauldix is keen to see it go in if you're OK with it. It looks good to me. Merge? A resolve is obviously necessary.

**Pull request merged**